src/README.md

```diff
@@ -1,72 +0,0 @@
-## grammY Commands Plugin
```

Comment:
why are we removing the README?

Reply:
It's is removing it from the `src` folder. It was dupe in both the `root` and `src` dir's

Reply:
That is necessary because when publishing to `deno.land/x`, only `src/` is published there. We therefore need a small README file in that directory. All it needs to do is to link to the main repo. Look how it's done in the core lib: https://git.io/grammyjs

Reply:
corrected in [275d2cb](https://github.com/grammyjs/commands/pull/31/commits/275d2cb55bfbc8b31e2a9170b1f5698010c221fa)